chore(explorer): append base world ABI to API endpoint (#3653)
latticexyz/mud: .changeset/eleven-crabs-jump.md

---
"@latticexyz/explorer": patch
---

> World ABI endpoint is now extended with base world ABI.

Reviewer: ```suggestion
World ABI endpoint is now includes the base world ABI.
```
